src/lib/openapi/index.ts

```diff
@@ -124,6 +126,12 @@ export interface ClientApiOperation
     tags: ['client'];
 }
 
+export interface OtherApiOperation
+    extends Omit<OpenAPIV3.OperationObject, 'tags'> {
+    operationId: string;
+    tags: ['other'];
```

Comment:
Does something like this make sense given that these are 'other' controllers?

Reply:
Yeah, good idea! We can probably generalize the type a bit, if we want: 

```ts
interface ApiOperation<Tag = 'client' | 'admin' | 'other'>
    extends Omit<OpenAPIV3.OperationObject, 'tags'> {
    operationId: string;
    tags: [Tag];
}

export type AdminApiOperation = ApiOperation<'admin'>;
export type ClientApiOperation = ApiOperation<'client'>;
export type OtherApiOperation = ApiOperation<'other'>;
```